src/analysis/souffle/epsilon_analysis.dl

```diff
@@ -0,0 +1,77 @@
+#ifndef SRC_ANALYSIS_SOUFFLE_EPSILON_ANALYSIS_DL_
+#define SRC_ANALYSIS_SOUFFLE_EPSILON_ANALYSIS_DL_
+
+#include "src/analysis/souffle/sensitivity_analysis.dl" 
+
+#define GLOBAL_EPSILON_OPERATOR as("global_epsilon", Operator)
+#define PRIVACY_MECHANISM_OPERATOR as("privacy_mechanism", Operator)
+
+// Global epsilon is a value determined by users of Raksha. 
+// We could eventually set one as the default. 
+.decl isGlobalEpsilon(epsilon: number)
+isGlobalEpsilon(epsilon) :- isOperation(operation), 
+    operationHasOperator(operation, GLOBAL_EPSILON_OPERATOR),
+    operationHasAttribute(operation, ["epsilon", $NumberAttributePayload(epsilon)]).
```

Comment:
Do you mean have .input isGlobalEpsilon()? I went this route cause I thought isOperation is the only way to translate IR into datalog facts automatically.

Reply:
Hm, you raise a good point. It would probably be labor intensive to follow my suggestion. Also, it seems like the global epsilon is a temporary situation, so I think this is OK for now.